server/rest/src/test/java/org/infinispan/rest/resources/CacheResourceTest.java

```diff
@@ -86,12 +86,13 @@ protected void defineCaches(EmbeddedCacheManager cm) {
    public Object[] factory() {
       return new Object[]{
             new CacheResourceTest().withSecurity(false),
-            new CacheResourceTest().withSecurity(true),
+            new CacheResourceTest().withSecurity(true).http2(true),
```

Comment:
security and ssl tests with HTTP1 missing?

Reply:
ps. `org.infinispan.rest.resources.CacheResourceOffHeapTest` be changed as well to test all combination (hopefully not an overkill)

Reply:
I will add https and  http1. I though it'd be rare, since we usually offer http/2 after the protocol negotiation during the SSL handshake, but you never know what are the clients out there. And Offheap + any combination is not necessary I think. It will bloat the test suite